<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFeedbackDialog</class>
 <widget class="QDialog" name="UserFeedbackDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Any Feedback Is Welcome</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Getting frustrated by NeuTu? 
Please provide feedbacks/complaints below and send them to the developers. 
The message will ONLY be checked by NeuTu developers to determine 
the priority of future improvements. 
You can send the same comment again and again to emphasize your need. </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="feedbackTextEdit">
     <property name="placeholderText">
      <string>Type any comments here, e.g. I hate to remember too many shortcuts.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="anonymousCheckBox">
     <property name="toolTip">
      <string>Send the message without revealing your identity.
But if you expect follow-up from the developers, please check it off.</string>
     </property>
     <property name="text">
      <string>Anonymous</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserFeedbackDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserFeedbackDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
